Tree/PhpcrOdmTree.php

```diff
@@ -127,6 +127,33 @@ public function __construct(
 
         $this->depth = $options['depth'];
         $this->preciseChildren = $options['precise_children'];
+
+        if (!$this->assetHelper instanceof CoreAssetsHelper) {
+            if (!(class_exists('\Symfony\Bundle\FrameworkBundle\Templating\Helper\AssetsHelper')
+                    && $this->assetHelper instanceof \Symfony\Bundle\FrameworkBundle\Templating\Helper\AssetsHelper
+                )
+                && !(class_exists('\Symfony\Component\Asset\Packages')
+                    && $this->assetHelper instanceof \Symfony\Component\Asset\Packages
+                )
+                && !(interface_exists('\Symfony\Component\Asset\PackageInterface')
+                    && $this->assetHelper instanceof \Symfony\Component\Asset\PackageInterface
```

Comment:
afaik instanceof does not care whether the target class/interface exists. if i am not mistaken, you could simplify this code by just checking instanceof.

can you add a little bit of documentation explaining which symfony version needs what? its crazy that we have 4 different classes/interfaces that could be passed.